### PR TITLE
Add ability to upload data of new client and session or existing client with new session

### PR DESCRIPTION
**Pull request by ZaxAtax**

### CODE REVIEW

File: frontend/src/components/session-grid/SessionForms.svelte [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
```
Comment (john-rice) [inline frontend/src/components/session-grid/SessionForms.svelte:1]: In avoiding ambiguity,  always prefer more descriptive names. "session" and "client" can refer to different things. Other naming can be considered such as booking, appointment, photoSession, shoot, etc lend to avoiding ambiguity.

File: frontend/src/components/session-grid/SessionForms.svelte [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      clients = data;
+    }
+  };
+
+  onMount(() => {
+    getData();
+  });
+</script>
+
+<h2>Create New Client/Session</h2>
```
Comment (john-rice) [inline frontend/src/components/session-grid/SessionForms.svelte:67]: Is this content too specific for an "astro starter" repo or a good fit for a default example?

**Review by dman926 — APPROVED** [review]
LGTM

File: frontend/src/components/session-grid/SessionForms.svelte (hunk 1 above)
Comment (dman926) [reply frontend/src/components/session-grid/SessionForms.svelte]: These were the names of the existing spreadsheets. We renamed "session" to "appointment". "client" can stay as is.

File: frontend/src/components/session-grid/SessionForms.svelte (hunk 2 above)
Comment (dman926) [reply frontend/src/components/session-grid/SessionForms.svelte]: A photography site is a pretty good default.

**Pull request merged**